go.mod

```diff
@@ -16,11 +16,10 @@ require (
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.5.1
-	golang.org/x/net v0.0.0-20200226121028-0de0cce0169b // indirect
-	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 // indirect
 	k8s.io/api v0.18.2
 	k8s.io/apimachinery v0.18.6
 	k8s.io/client-go v0.18.2
 	sigs.k8s.io/controller-runtime v0.6.0
 	sigs.k8s.io/controller-tools v0.3.1-0.20200716001835-4a903ddb7005 // indirect
+	sigs.k8s.io/kind v0.8.1 // indirect
```

Comment:
Please remove this and the changes to `go.sum`. This happened because KinD is installed via `go get` in the scripts.